compiler/util.cc

```diff
@@ -1,9 +1,8 @@
 #include "compiler/util.h"
 
-#include <locale>
-
 #include <common/strutil.h>
 #include <compiler/tensor.h>
+#include <cctype>
```

Comment:
Please move this (and locale) before `common/strutil.h`

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes